index.js

```diff
@@ -29,34 +29,57 @@ process.title = 'logdna-agent'
 commander._name = 'logdna-agent'
 commander
   .version(pkg.version, '-v, --version')
-  .description('This agent collect and ship logs for processing. Defaults to /var/log if run without parameters.')
-  .option('-c, --config <file>', `uses alternate config file (default: ${config.DEFAULT_CONF_FILE})`)
+  .description(
+    'This agent collect and ship logs for processing.'
+      + 'Defaults to /var/log if run without parameters.'
+  )
+  .option(
+    '-c, --config <file>'
+  , `uses alternate config file (default: ${config.DEFAULT_CONF_FILE})`
```

Comment:
Let's be consistent with capitalization.  Some of these sound like sentences, so I'm a fan of first letter capital.

Reply:
btw, I know you didn't write any of these :)

Reply:
Thanks for the look out. I know there were some legacy things so I appreciate you going through and seeing where things are wrong.